src/bootstrap/src/utils/cc_detect.rs

```diff
@@ -287,19 +330,22 @@ pub(crate) enum Language {
 }
 
 impl Language {
-    /// Obtains the name of a compiler in the GCC collection.
+    /// Returns the executable name for a GCC compiler corresponding to this language.
     fn gcc(self) -> &'static str {
         match self {
             Language::C => "gcc",
             Language::CPlusPlus => "g++",
         }
     }
 
-    /// Obtains the name of a compiler in the clang suite.
+    /// Returns the executable name for a clang-based compiler corresponding to this language.
```

Comment:
this diff doesn't seem to add anything

Reply:
I liked the old one more tbh. Especially "this language" part doesn't seem good.